src/components/About/About.tsx

```diff
@@ -1,16 +1,30 @@
 import { LinkButton } from "../LinkButton/LinkButton";
-import { ContentWrapper, Subtitle, Title, Wrapper } from "./About.styles";
 
 export const AboutSection = ({ header }: { header: { fields: any } }) => {
-  const { id, title, subtitle } = header.fields;
+  const { id, title, subtitle, thumb } = header.fields;
 
   return (
-    <Wrapper id={id}>
-      <ContentWrapper>
-        <Title>{title}</Title>
-        <Subtitle>{subtitle}</Subtitle>
-        <LinkButton href="/">Saiba mais</LinkButton>
-      </ContentWrapper>
-    </Wrapper>
+    <section id={id} className="w-full bg-grey-100 py-8 lg:py-12">
+      <div className="w-full max-w-[1440px] mx-auto px-4 flex flex-col gap-6">
+        <h2 className="text-[30px] leading-[36px] tracking-[-0.0075em] font-bold text-left lg:text-left text-center">
+          {title}
+        </h2>
+        <div className="flex flex-col lg:flex-row gap-6">
+          <img
```

Comment:
Instead of using html's <img /> tag, it's better for performance if we use next's <Image /> component, since that with it we can set the width and height the browser will fetch the image